README.md

```diff
@@ -113,6 +115,10 @@ github iosphere/ISHHoverBar
 See the [Carthage repository](https://github.com/Carthage/Carthage) to get started
 with Carthage.
 
+## More OpenSource projects by iosphere
+
+* [`ISHPermissionKit`](https://github.com/iosphere/ISHPermissionKit) - A polite and unified way of asking for permission on iOS
```

Comment:
Can we display the icons here, similar to the "Apps using permission kit" section?